### ISSUE TITLE
Is there a way to create a word cloud in the shape of one or multiple letters ?

### ISSUE THREAD

**Issue by lazywinadmin** [vexx32/PSWordCloud#17]
For example a word cloud with the "PowerShell" word which would contains all the cmdlet names present on your machine.

Or "PSGallery" with the top 200 modules names

**Comment by vexx32** [comment]
Currently, no, not without modifying the code.

Doing so wouldn't be _trivial_, but it wouldn't be overly tricky. I wouldn't recommend doing this in a bitmap image, however; the detail level is not high enough.

I have an experimental branch implementing the same code in C# in the `Cmdlet` branch that is almost completely functional. We could look at adding that as a new feature there, as SkiaSharp is used instead & supports vector files, which would make this significantly more feasible.

**Comment by lazywinadmin** [comment]
Thanks @vexx32 for the quick reply! Not sure i can help with the c# but I'm available if you need me to test it out 😄

**Comment by vexx32** [comment]
Yeah, I'll have to tidy it up and at least put a pre-release onto the gallery or something.

Not 100% convinced of its stability just yet, but it's very consistent so far. I'll give you a ping when we get a pre-release out if you'd like to put it through its paces :D

**Comment by vexx32** [comment]
@lazywinadmin I've released the cmdlet version to the gallery.

I have some ideas on how to get started on this, and of some potential obstacles... I'll have a chat with some of the SkiaSharp fellows to see if there's a good way through some of the trickier points. 😄
